plugins/scanners/fossid/src/test/kotlin/FossIdTest.kt

```diff
@@ -935,6 +935,45 @@ class FossIdTest : WordSpec({
             }
         }
 
+        "apply exclusion rules to a origin scan" {
+            val projectCode = projectCode(PROJECT)
+            val scanCode = scanCode(PROJECT, FossId.DeltaTag.ORIGIN)
```

Comment:
You need a scancode without the delta part (no really critical but consistency).

Reply:
Had a look at other non-delta scans and modified the code in the same way.